infrastructure/main.tf

```diff
@@ -1 +1,26 @@
-provider "azurerm" {}
\ No newline at end of file
+provider "azurerm" {}
+
+locals {
+  ase_name            = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
+  is_preview          = "${(var.env == "preview" || var.env == "spreview")}"
+  local_env           = "${local.is_preview ? "aat" : var.env}"
+  s2s_rg              = "rpe-service-auth-provider-${local.local_env}"
+  s2s_url             = "http://${local.s2s_rg}.service.core-compute-${local.local_env}.internal"
+  ccd_api_url         = "http://ccd-data-store-api-${local.local_env}.service.core-compute-${local.local_env}.internal"
```

Comment:
so sad this service needs the dependency on this one 😢

Reply:
will remove this for now as we don't know how we are going to implement it